TorchSharp/TypeGeneration.tt

```diff
@@ -198,24 +198,23 @@ foreach (var type in new [] {
         {
             handle = TH<#=tname#>Tensor_new ();
         }
-        
-        internal <#=tname#>Tensor (HType handle)
+
+                internal <#=tname#>Tensor (HType handle)
```

Comment:
why do we indent like that?

Reply:
No idea. VS added tabs instead of spaces although I changed in my default settings :( Anyway fixed